dcc-submission-ui/dictionary/app/scripts/helpers/dictionary-utils.js

```diff
@@ -1,4 +1,4 @@
-/* globals $ */
+/* globals $,jsondiffpatch */
```

Comment:
JSHint gives error if I don't do this. There are multiple occasions this type of ignoring is being done in dictionary viewer app.